yt_dlp/extractor/radiofrance.py

```diff
@@ -102,3 +144,327 @@ def _real_extract(self, url):
+            'thumbnail': r're:^https?://.*\.(?:jpg|png)',
+        },
+        'playlist_mincount': 321,
+    }, {
+        'url': 'https://www.radiofrance.fr/franceinter/podcasts/le-7-9',
+        'only_matching': True,
+    }, {
+        'url': 'https://www.radiofrance.fr/mouv/podcasts/dirty-mix',
+        'only_matching': True,
+    }]
+
+    _METADATA_KEY = 'expressions'
+
+    def _generate_playlist_entries(self, podcast_id, podcast_response):
+        for page_num in itertools.count(2):
+            for episode in podcast_response['items']:
+                yield self.url_result(
+                    f'https://www.radiofrance.fr/{episode["path"]}', FranceCultureIE, **traverse_obj(episode, {
+                        'title': 'title',
+                        'description': 'standFirst',
+                        'timestamp': 'publishedDate',
+                        'thumbnail': ('visual', 'src'),
+                    }))
+
+            next_cursor = podcast_response['next']
+            if not next_cursor:
+                break
+
+            podcast_response = self._download_json(
+                f'https://www.radiofrance.fr/api/v2.1/concepts/{podcast_id}/expressions', podcast_id,
+                note=f'Downloading page {page_num}', query={'pageCursor': next_cursor})
+
+
+class RadioFranceProfileIE(RadioFrancePlaylistBase):
+    _VALID_URL = rf'{RadioFranceBase._VALID_URL_BASE}/personnes/(?P<id>[\w-]+)'
+
+    _TESTS = [{
+        'url': 'https://www.radiofrance.fr/personnes/thomas-pesquet?p=3',
+        'info_dict': {
+            'id': '86c62790-e481-11e2-9f7b-782bcb6744eb',
+            'display_id': 'thomas-pesquet',
+            'title': 'Thomas Pesquet',
+            'description': 'Astronaute à l\'agence spatiale européenne',
+        },
+        'playlist_mincount': 212,
+    }, {
+        'url': 'https://www.radiofrance.fr/personnes/lea-salame',
+        'only_matching': True,
+    }]
+
+    _METADATA_KEY = 'documents'
+
+    def _generate_playlist_entries(self, profile_id, profile_response):
+        for page_num in itertools.count(2):
+            for entry in profile_response['items']:
+                yield self.url_result(
+                    f'https://www.radiofrance.fr/{entry["path"]}', **traverse_obj(entry, {
+                        'title': 'title',
+                        'description': 'standFirst',
+                        'timestamp': 'publishedDate',
+                        'thumbnail': ('visual', 'src'),
+                    }))
+
+            next_cursor = profile_response['pagination']['next']
+            if not next_cursor:
+                break
+
+            profile_response = self._download_json(
+                f'https://www.radiofrance.fr/api/v2.1/taxonomy/{profile_id}/documents', profile_id,
+                note=f'Downloading page {page_num}', query={
+                    'relation': 'personality',
+                    'cursor': next_cursor
+                })
```

Comment:
This is the only part that is different. You could do something like this and move rest to base class:
```py
    def _call_api(self, profile_id, cursor, page_num):
        resp = self._download_json(...)
        resp['next'] = traverse_obj(resp, ('pagination', 'next'))
        return resp
```